src/parser/demonhunter/havoc/modules/Abilities.js

```diff
@@ -42,9 +42,9 @@ class Abilities extends CoreAbilities {
           base: 1500,
         },
         castEfficiency: {
-          suggestion: combatant.hasTalent(SPELLS.FIRST_BLOOD_TALENT.id),
-          recommendedEfficiency: 0.95,
-          extraSuggestion: <>This should be part of your single target rotation due to the <SpellLink id={SPELLS.FIRST_BLOOD_TALENT.id} /> talent. This includes the <SpellLink id={SPELLS.DEATH_SWEEP.id} /> casts since they are the same ability and share their cooldowns.</>,
+          suggestion: combatant.hasTalent(SPELLS.FIRST_BLOOD_TALENT.id) || combatant.hasTalent(SPELLS.TRAIL_OF_RUIN_TALENT.id),
+          recommendedEfficiency: 0.85,
+          extraSuggestion: <>This should be part of your single target rotation due to using either or both of the <SpellLink id={SPELLS.FIRST_BLOOD_TALENT.id} /> and <SpellLink id={SPELLS.TRAIL_OF_RUIN_TALENT.id} /> talent. This includes the <SpellLink id={SPELLS.DEATH_SWEEP.id} /> casts since they are the same ability and share their cooldowns.</>,
```

Comment:
This sentence is strange to me. Just reads weird

Reply:
Could just remove the text about the talents tbh if its makes it weird. But I feel the need to say its because you are using FIRST_BLOOD_TALENT **and/or** TRAIL_OF_RUIN_TALENT, could'nt find a better way to do it than saying "using either or both of the..." and not just point at First Blood as they did before.

But I can look into it.

Reply:
![image](https://user-images.githubusercontent.com/4808216/103505976-f1c2d480-4e5b-11eb-82d0-e0f43b78ba75.png)

This is the best I can come up with. Im not the best with sentences.